pkg/solana/config/config_test.go

```diff
@@ -15,18 +15,26 @@ import (
 
 // testing configs
 var (
-	testBalancePoll            = models.MustMakeDuration(1 * time.Minute)
-	testConfirmPeriod          = models.MustMakeDuration(2 * time.Minute)
-	testCachePeriod            = models.MustMakeDuration(3 * time.Minute)
-	testTTL                    = models.MustMakeDuration(4 * time.Minute)
-	testTxTimeout              = models.MustMakeDuration(5 * time.Minute)
-	testTxRetryTimeout         = models.MustMakeDuration(6 * time.Minute)
-	testTxConfirmTimeout       = models.MustMakeDuration(7 * time.Minute)
+	testBalancePoll            = mustDuration(1 * time.Minute)
+	testConfirmPeriod          = mustDuration(2 * time.Minute)
+	testCachePeriod            = mustDuration(3 * time.Minute)
+	testTTL                    = mustDuration(4 * time.Minute)
+	testTxTimeout              = mustDuration(5 * time.Minute)
+	testTxRetryTimeout         = mustDuration(6 * time.Minute)
+	testTxConfirmTimeout       = mustDuration(7 * time.Minute)
 	testPreflight              = false
 	testCommitment             = "finalized"
 	testMaxRetries       int64 = 123
 )
 
+func mustDuration(d time.Duration) utils.Duration {
```

Comment:
I was actually happy to remove it from exported API :laughing: . I wouldn't be opposed to a `package testutils` with a `MustDuration(*testing.TB,time.Duration)` that fails the test instead of panicing though... :thinking: 
Bigger picture, I am hoping that this odd custom wrapper type won't be necessary with the new TOML config, and this will dissapear.